core/modules/bypass.h

```diff
@@ -11,6 +11,7 @@ class Bypass final : public Module {
   static const gate_idx_t kNumIGates = MAX_GATES;
   static const gate_idx_t kNumOGates = MAX_GATES;
 
+  Bypass() { max_allowed_workers_ = Worker::kMaxWorkers; }
```

Comment:
It is a variable declared in the base class, and cannot be initialized in the initializer list of a derived class, see for [example](http://stackoverflow.com/questions/7405740/how-can-i-initialize-base-class-member-variables-in-derived-class-constructor).